integration-tests/sceptre-project/config/9/B.yaml

```diff
@@ -1,4 +1,4 @@
 template_path: dependencies/dependent_template_local_export.json
 region: eu-west-1
 parameters:
-  DependentStackName: !stack_output_external "{project_code}-9-A::StackName default::eu-west-1"
+  DependentStackName: !stack_output_external "sceptre-integration-tests-452379e268b711ecaf07acde48001122-9-A::StackName default::eu-west-1"
```

Comment:
You need to revert this back to the file in master. I accidentally do this all the time. When you run integration tests, there is at least one that seems to replace this value in your local files. Way too easy to commit it up, unfortunately.

Reply:
done